### PR TITLE
Rollback the grow spinners

### PR DESCRIPTION
**Pull request by louismaximepiton**
Get the grow spinners back to v5. Need a dark version ?

[### Preview]()

### CODE REVIEW

File: site/content/docs/5.1/components/spinners.md [hunk 1]
```diff
@@ -38,14 +44,39 @@ The border spinner uses `currentColor` for its `border-color`, meaning you can c
...
+Once again, this spinner is built with `currentColor`, so you can easily change its appearance with [text color utilities][color]. Here it is in blue, along with the supported variants.
+
+{{< example >}}
+<div class="spinner-grow text-primary" role="status">
+  <span class="visually-hidden">Loading...</span>
+</div>
+<div class="spinner-grow text-secondary" role="status">
+  <span class="visually-hidden">Loading...</span>
+</div>
+<div class="spinner-grow text-light" role="status">
```
Comment (louismaximepiton) [inline site/content/docs/5.1/components/spinners.md:76]: For `.text-light`, we override the background-color and it doesn't appear as white. Should we use bg-color utilities instead ?

Comment (julien-deramond) [reply site/content/docs/5.1/components/spinners.md]: IMO we can drop this example since it is not useful with a dark background

**Review by julien-deramond — CHANGES_REQUESTED** [review]
I'm proposing https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1041/commits/b49c5337d96898bd5cdde592ba02ddf0feefbd27 to enhance some parts of this PR. Please tell me if you're OK with it or revert it if not.

**Comment by louismaximepiton** [comment]
Those changes sounds great to me !

**Review by julien-deramond — APPROVED** [review]

**Pull request merged**